ops/roles/infra/tasks/main.yml

```diff
@@ -50,16 +50,6 @@
 - name: Enable user namespace cloning to allow running Chromium in a sandbox
   command: sysctl -w kernel.unprivileged_userns_clone=1
 
-# do it manually as it does not seem to work when deploying
```

Comment:
`gh` is used toi build the release and upload it to github.

Do you plan to put this doc somewhere else ?

Reply:
As discussed orally, dead code is removed, and we created this issue #700